lib/matplotlib/backends/_backend_gtk.py

```diff
@@ -62,6 +62,19 @@ def _create_application():
     return _application
 
 
+@functools.lru_cache()
```

Comment:
Do we really need a cache here and then again in GTK3? There wasn't one in GTK4...

Reply:
let's not bother to cache a dict lookup, indeed.